app/hooks/files.ts

```diff
@@ -49,32 +49,30 @@ const getFileInfo = async (serverUrl: string, bookmarks: ChannelBookmarkModel[],
     }
 };
 
-export const useImageAttachments = (filesInfo: FileInfo[], publicLinkEnabled: boolean) => {
+export const useImageAttachments = (filesInfo: FileInfo[]) => {
     const serverUrl = useServerUrl();
     return useMemo(() => {
         return filesInfo.reduce(({images, nonImages}: {images: FileInfo[]; nonImages: FileInfo[]}, file) => {
             const imageFile = isImage(file);
             const videoFile = isVideo(file);
-            if (imageFile || (videoFile && publicLinkEnabled)) {
+            if (imageFile || videoFile) {
```

Comment:
`publicLinkEnabled` was not adding any value, since the only difference would be that if public link is not enabled, we never use the local path, which seems wrong.